client/tests/register_test.py

```diff
@@ -90,6 +90,37 @@ def my_function(df):
     # Checks that Transformation definition does not error when converting to source
     dec.to_source()
 
+@pytest.mark.parametrize(
```

Comment:
@ahmadnazeri, I added my model registration tests here given this file already follows the patterns we discussed; however, I still think we could clean things up (e.g. reusable fixtures into `conftest.py` and standardize on arrangement, etc.).

Given this, if you think it's important to separate out the "newer" tests into their own files, I can move these to a new file (e.g. `test_model_registration.py`). Let me know what you think.

Reply:
no worries, this it good!

Reply:
Just noticed I moved this into `conftest.py` and never actually used it as I ended up moving my model serving tests to their own file and registering my primary source from the `/test_files/input_files`.

When I time comes to further refactor our existing tests, I'll definitely look into this builtin fixture.